scrapy/extensions/logstats.py

```diff
@@ -30,21 +30,25 @@ def from_crawler(cls, crawler):
     def spider_opened(self, spider):
         self.pagesprev = 0
         self.itemsprev = 0
-
+        self.response_bytesprev = 0.0
```

Comment:
:lipstick: I think `self.responsebytesprev` would be more in line with the existing code, even if snake case is more Pythoninc.

```suggestion
        self.response_bytesprev = 0.0
```